classes/shortcodes.php

```diff
@@ -91,8 +92,10 @@ function do_shortcode( $args ) {
 				'after'             => '',
 				'map_width'         => '75px',
 				'map_height'        => '75px',
-				'default_lat'       => get_post_meta( $post->ID, 'location_lat', true ),
-				'default_lng'       => get_post_meta( $post->ID, 'location_lng', true ),
+				'default_lat'       => get_post_meta( $post->ID, 'location_lat',
+					true ),
+				'default_lng'       => get_post_meta( $post->ID, 'location_lng',
```

Comment:
__[code_sniffer]__
* Spaces must be used to indent lines; tabs are not allowed
* Opening parenthesis of a multi-line function call must be the last content on the line
* Only one argument is allowed per line in a multi-line function call

<!-- Comment by [SideCI](https://sideci.com) -->